PubSubSpec.md

```diff
@@ -22,24 +24,36 @@ This section describes the functions performed at data publishers.
 
 ### Segmentation and Encapsulation
 
-Data blobs received from the `publish` call MUST be segmented if larger than the network MTU. Each segment is named per the NDN segmentation guidelines, with version number of `0`. Each segment MUST contain the `FinalBlockId` field set to the name of the last segment. Segmentation MAY be done asynchronously.
+Data blob received in the `publish` call MUST be segmented if it is larger than the network MTU. It SHOULD NOT be segmented if it is small enough to fit directly in the network MTU.
```

Comment:
* The plural "blobs" was fine here... if you want the singular add the article: "the data blob"

* It took me a while to understand what "received" meant in this context... would "passed to" be better?